### DISCUSSION TITLE
My application has not approved

### DISCUSSION THREAD

**Discussion by ShadowVik** [community/community#50816] [GitHub Education]
I tried to submit and change even my name and bio, but it still doesn't work and I get the same email response.

Suggested proof to expedite your approval:
You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

What should I do now?

**Comment by samus-aran** [comment]
Hi @ShadowVik 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Hope this helps! Let us know how you get along 👍

**Comment by ShadowVik** [comment]
What if I don't want to display my personal information publicly? Why I can't just prove who I am without being forced to share my name and other stuff with everyone? 
And I even put all the information, and it got rejected. It's just annoying, and I don't see any point in it. 
Thank you for reply.
